src/org/labkey/test/components/ui/files/FileAttachmentContainer.java

```diff
@@ -0,0 +1,249 @@
+package org.labkey.test.components.ui.files;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.html.FileInput;
+import org.labkey.test.components.html.Input;
+import org.openqa.selenium.ElementNotInteractableException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.io.File;
+import java.util.List;
+
+/*
+    Wraps https://github.com/LabKey/labkey-ui-components/blob/fb_assayResultsFiles/packages/components/src/internal/components/files/FileAttachmentContainer.tsx
+
+ */
+public class FileAttachmentContainer extends WebDriverComponent<FileAttachmentContainer.ElementCache>
+{
+    private final WebElement _el;
+    private final WebDriver _driver;
+
+    public FileAttachmentContainer(WebElement element, WebDriver driver)
```

Comment:
Leaving the c'tor public allows calling code to specify the componentElement itself- this follows the pattern set in `FileUploadField`, and it happens that this component works in cases FileUploadField does (they can be interchangeably used in the compact-mode usages like in `EntityBulkUpdateDialog` )